Assets/GameAssets/Player/PlayerScripts/Harpoon/HarpoonProjectileMovement.cs

```diff
@@ -66,4 +67,16 @@ private void HarpoonFiredProjectileMovement(Vector3 movement)
     {
         transform.position += movement;
     }
+
+    /// <summary>
+    /// Used to destroy the harpoon projectile when it hits something in scene.
+    /// </summary>
+    /// <param name="block"></param> what it collides against. 
+    private void OnTriggerEnter(Collider block)
+    {
+        if(block.gameObject.name != "WeakPoint(Clone)")
```

Comment:
use block.gameobject.tag != "Whatever weakpoint's tag is" instead so yo don't have to worry too much about clones